leetcode/hard/162._Find_Peak_Element.py

```diff
@@ -0,0 +1,28 @@
+# Link to problem: https://leetcode.com/problems/find-peak-element/description/
+# Loom explaining solution you wrote (5m):
+# Solution:
+# Description
+# Key ideas
+# TODO
+# Difficulty: Hard
```

Comment:
This is listed on LC as medium.

Did you list this as Hard because you thought it was challenging? Or did you just write it wrong from LC?

Reply:
Difficulty here denotes how I evaluate the problem, this is added as I find a lot of medium difficulty LC questions easy/ trivial.